### ISSUE TITLE
Final Step

### ISSUE THREAD

**Issue by existeundelta** [gl-lamhnguyen/s3-glacier-tools#1]
Is it posible that now 

--query "CopyObjectResult.{LastModified}"

is not part of the command?

thanks!

**Comment by existeundelta** [comment]
if i drop it i get
An error occurred (InvalidRequest) when calling the CopyObject operation: This copy request is illegal because it is trying to copy an object to itself without changing the object's metadata, storage class, website redirect location or encryption attributes.

**Comment by existeundelta** [comment]
it seems now is
--metadata-directive REPLACE

**Comment by casper-gh** [comment]
This `--query "CopyObjectResult.{LastModified}"` is a part of aws cli command in general, basically it filter the json result. Unless the response json has a different format, it should work. But anyway thanks for your commit, I'll check when I have time.

**Comment by existeundelta** [comment]
because was not absolute sure, I was waiting if you confirm it, you have more experience, 
thanks

**Comment by casper-gh** [comment]
I just checked the script again, that error means there is nothing to do on the file, so basically skip it. Adding `metadata-directive` will ignore error and copy the file, which takes more time.

**Comment by existeundelta** [comment]
finally i used it with --metadata-directive REPLACE, that as you say toke a lot of time, but i don't understand "that error means there is nothing to do on the file, so basically skip it. " because in may case after step 2  (recover) files still were in glacier
